backend/shop/models.py

```diff
@@ -12,6 +12,7 @@ def __str__(self) -> str:
 
 class Order(models.Model):
     products = models.ManyToManyField("Product", through="OrderItem")
+    buyer = models.ForeignKey(ShopProfile, on_delete=models.CASCADE, null=False)
```

Comment:
https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.null

null=False, is already default. No need to specify it.